Bio/SeqUtils/CodonUsage.py

```diff
@@ -106,24 +105,18 @@ def generate_index(self, fasta_file):
         # now to calculate the index we first need to sum the number of times
         # synonymous codons were used all together.
         for aa in SynonymousCodons:
-            total = 0.0
-            # RCSU values are CodonCount/((1/num of synonymous codons) * sum of
-            # all synonymous codons)
-            rcsu = []
             codons = SynonymousCodons[aa]
 
-            for codon in codons:
-                total += self.codon_count[codon]
+            count_max = max((self.codon_count[codon] for codon in codons))
```

Comment:
How about this, or could there be additional values in the dictionary not in the the codon list?

```python
count_max = max(self.codon_count.values())
```

Reply:
Thank you for the suggestion, Peter! It is implemented.